bindings/wasm/src/did/wasm_document.rs

```diff
@@ -107,6 +239,18 @@ impl WasmDocument {
   // Verification Methods
   // ===========================================================================
 
+  /// Returns a list of all {@link VerificationMethod} in the DID Document.
+  #[wasm_bindgen(js_name = methods)]
```

Comment:
Nit: unnecessary `js_name`.

```suggestion
  #[wasm_bindgen]
```